CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
-
+## 2.6.0 (2021-04-26)
+### Changed
+- Decorate metrics with `mongodbClusterName`
+  `clusterName` (deprecated) attribute might get overwritten by the Kubernetes integration when running inside kubernetes.
```

Comment:
isn't the agent that overwrites the clusterName?

Reply:
yes is true, i'll change that thanks.